_data/cryptocurrencies.yml

```diff
@@ -301,6 +301,12 @@ websites:
       - totp
     doc: https://support.coinjar.com/hc/en-us/articles/202910075
 
+  - name: CoinLoan
+    url: https://coinloan.io/
+    img: coinloan.svg
+    tfa:
+      - totp
```

Comment:
```suggestion
      - totp
    doc: https://medium.com/coinloan/securing-your-data-with-two-factor-authentication-2fa-43a3546633a
```

Reply:
I found a copy of that post on [their blog](https://blog.coinloan.io/securing-your-accounts-with-two-factor-authentication-2fa/) so decided that was a better fit.